common/flatpak-installation.c

```diff
@@ -1519,6 +1519,40 @@ flatpak_installation_get_config (FlatpakInstallation *self,
   return flatpak_dir_get_config (dir, key, error);
 }
 
+/**
+ * flatpak_installation_get_min_free_space_bytes:
+ * @self: a #FlatpakInstallation
+ * @out_bytes: (out): Location to store the result
+ * @error: Return location for a #GError
+ *
+ * A convenience wrapper around ostree_repo_get_min_free_space_bytes().
+ * This can be used to query the min-free-space-* config option of the
+ * repo by clients like gnome-software.
```

Comment:
I think @matthiasclasen's point at https://github.com/flatpak/flatpak/commit/04b9e5af0afbba0cf5fd42e9b4b5c921f951795e#r31038814 still stands:

> The documentation should tell me what the function does and what i would want to use it for. What are apps supposed to do with this number ?

Reply:
(There's a lot of rationale in the commit message, but that's not what API users will see.)

Reply:
One question I still have is: why is this value stored in ostree config - is ostree using it for its own purposes ?
If so, why do clients have to do their own manual computation ?

Reply:
Can we change the documentation to something like:

Returns the min-free-space config value from the OSTree repository of this installation.

Applications can use this value, together with information about the available disk space and
the size of pending updates/installs, to estimate whether a pull operation will fail due to running
out of disk space.